flaml/tune/searcher/blendsearch.py

```diff
@@ -26,7 +28,7 @@
 from .flow2 import FLOW2
 from ..space import add_cost_to_space, indexof, normalize, define_by_run_func
 from ..result import TIME_TOTAL_S
-
+from ..utils import get_lexico_bound

```

Comment:
get_lexico_bound could be included in this file as a staticmethod of `BlendSearch`